source/_docs/z-wave/controllers.markdown

```diff
@@ -1,18 +1,18 @@
 ---
-title: "Z-Wave Controllers"
-description: "Extended instructions how to setup Z-Wave."
+title: "Z-Wave adapters"
+description: "Extended instructions how to set up Z-Wave."
 related:
   - docs: /integrations/zwave_js/
     title: Z-Wave integration
 ---
 
-## Supported Z-Wave USB Sticks & Hardware Modules
+## Supported Z-Wave adapters
 
-You need to have a compatible Z-Wave stick or module installed. The following devices have been confirmed to work with Z-Wave JS:
+You need to have a compatible Z-Wave adapter installed. The following devices have been confirmed to work with Z-Wave JS:
 
 {% warning %}
 
-The firmwares of 700 and 800 series Z-Wave controllers have several bugs which impact the stability of the mesh and can cause the controller to become unresponsive. Because there is no known firmware version that is completely fixed, it is recommended to choose a firmware based on the following criteria:
+The firmware of 700 and 800 series Z-Wave adapters have several bugs which impact the stability of the mesh and can cause the adapter to become unresponsive. Because there is no known firmware version that is completely fixed, it is recommended to choose a firmware based on the following criteria:
```

Comment:
```suggestion
The firmwares of 700 and 800 series Z-Wave adapters have several bugs which impact the stability of the mesh and can cause the adapter to become unresponsive. Because there is no known firmware version that is completely fixed, it is recommended to choose a firmware based on the following criteria:
```
I think the plural is correct, since there are several firmwares that are affected. And 700 and 800 series generally have different firmwares.